src/functionalTest/java/uk/gov/hmcts/reform/blobrouter/FunctionalTestBase.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.hmcts.reform.blobrouter;
+
+import com.azure.storage.blob.BlobServiceClient;
+import com.azure.storage.blob.BlobServiceClientBuilder;
+import com.azure.storage.common.StorageSharedKeyCredential;
+import org.apache.commons.codec.binary.Hex;
+import org.apache.commons.codec.digest.DigestUtils;
+import org.junit.jupiter.api.BeforeEach;
+import uk.gov.hmcts.reform.blobrouter.config.TestConfiguration;
+
+import java.util.concurrent.ThreadLocalRandom;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.blobrouter.storage.StorageHelper.blobExists;
+
+public abstract class FunctionalTestBase {
+
+    protected TestConfiguration config;
+
+    protected BlobServiceClient blobRouterStorageClient;
+
+    @BeforeEach
+    void setUp() {
+        this.config = new TestConfiguration();
```

Comment:
Would be nice to load configuration only once

Reply:
Will make a static instance, then.

Reply:
done